internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -0,0 +1,248 @@
+	}
+}
+
+// newIPAddress returns client's public ip
+func (opts *Opts) newIPAddress() (string, error) {
+	command := "curl ifconfig.me"
+	cmd := exec.Command("bash", "-c", command)
+	stdout, err := cmd.Output()
+
+	if err != nil {
+		return "", errors.New("error in finding your public IP, please use --ip to provide your public ip")
+	}
+
+	return string(stdout), nil
+}
+
+func (opts *Opts) newWhitelist() (*atlas.ProjectIPWhitelist, error) {
+	whitelist := &atlas.ProjectIPWhitelist{
+		GroupID: opts.ConfigProjectID(),
+		Comment: accessListComment,
+	}
+
+	if opts.ipAddress == "" {
+		ipAddress, err := opts.newIPAddress()
+		if err != nil {
+			return nil, err
+		}
+		whitelist.IPAddress = ipAddress
+	}
+
+	return whitelist, nil
+}
+
+func (opts *Opts) newCluster() *atlas.Cluster {
+	diskSizeGB := float64(diskSizeGB)
+	return &atlas.Cluster{
+		GroupID:             opts.ConfigProjectID(),
+		ClusterType:         replicaSet,
+		ReplicationSpecs:    []atlas.ReplicationSpec{opts.newReplicationSpec()},
+		ProviderSettings:    opts.newProviderSettings(),
+		MongoDBMajorVersion: mdbVersion,
+		DiskSizeGB:          &diskSizeGB,
+		Name:                opts.clusterName,
+	}
+}
+
+func (opts *Opts) newReplicationSpec() atlas.ReplicationSpec {
+	var (
+		readOnlyNodes int64 = 0
+		priority      int64 = 7
+		shards        int64 = shards
+		members       int64 = members
+	)
+	replicationSpec := atlas.ReplicationSpec{
+		NumShards: &shards,
+		ZoneName:  zoneName,
+		RegionsConfig: map[string]atlas.RegionsConfig{
+			opts.region: {
+				ReadOnlyNodes:  &readOnlyNodes,
+				ElectableNodes: &members,
+				Priority:       &priority,
+			},
+		},
+	}
+	return replicationSpec
+}
+
+func (opts *Opts) newProviderSettings() *atlas.ProviderSettings {
+	return &atlas.ProviderSettings{
+		InstanceSizeName: tier,
+		ProviderName:     opts.provider,
+		RegionName:       opts.region,
+	}
+}
+
+// mongocli atlas dbuser(s) quickstart [--clusterName clusterName] [--provider provider] [--region regionName] [--projectId projectId] [--username username] [--password password]
+func Builder() *cobra.Command {
+	opts := &Opts{}
+	cmd := &cobra.Command{
+		Use: "quickstart",
+		Example: `mongocli atlas quickstart
+mongocli atlas quickstart --clusterName Test --provider GPC --username dbuserTest --password Test!
+`,
+		Short: QuickStart,
+		Long:  LongQuickStart,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore,
+				opts.InitOutput(cmd.OutOrStdout(), ""),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.clusterName, flag.ClusterName, "GetStarted", usage.ClusterName)
+	cmd.Flags().StringVar(&opts.provider, flag.Provider, "AWS", usage.Provider)
+	cmd.Flags().StringVarP(&opts.region, flag.Region, flag.RegionShort, "US_EAST_1", usage.Region)
+	cmd.Flags().StringVar(&opts.ipAddress, flag.IP, "", usage.AccessListIPEntry)
```

Comment:
[nit] it would be great to give the option to input this in the wizard if empty or use current IP